server/schedule/region_scatterer.go

```diff
@@ -238,7 +240,12 @@ func (r *RegionScatterer) scatterRegions(regions map[uint64]*core.RegionInfo, fa
 				continue
 			}
 			if op != nil {
-				ops = append(ops, op)
+				if ok := r.opController.AddOperator(op); !ok {
```

Comment:
Maybe we can enlarge the expiration time for the scatter region? Do we test it with this change?

Reply:
Enlarging the expiration time is s an optional solution. But I think it could affect other scheduling operations in a large cluster, so I chose this scheme which would only affect Scatter. I will add a test for my solution soon.